core/cmd/local_client.go

```diff
@@ -73,6 +75,24 @@ func loggedStop(app chainlink.Application) {
 	logger.WarnIf(app.Stop())
 }
 
+func checkFilePermissions(directory string) {
+	invalidPermissions := os.FileMode(0077)
```

Comment:
Using constants with comment for the permission masks(?) would make the code easier to reason about.